builtin/logical/pki/path_manage_keys.go

```diff
@@ -263,10 +263,13 @@ func (b *backend) pathImportKeyHandler(ctx context.Context, req *logical.Request
 	var pemBlock *pem.Block
 
 	var keys []string
+	blockCounter := 0
 	for len(bytes.TrimSpace(pemBytes)) > 0 {
+		blockCounter++
 		pemBlock, pemBytes = pem.Decode(pemBytes)
 		if pemBlock == nil {
-			return logical.ErrorResponse("provided PEM block contained no data"), nil
+			msg := fmt.Sprintf("error when parsing block %d: invalid PEM data", blockCounter)
```

Comment:
@klaus-sap `"fmt"` needs to be imported at the top of the file :-)

Reply:
done